versions/3.1.0.md

```diff
@@ -2675,9 +2674,14 @@ components:
 
 #### <a name="discriminatorObject"></a>Discriminator Object
 
-When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object can be used to aid in serialization, deserialization, and validation.  The discriminator is a specific object in a schema which is used to inform the consumer of the document of an alternative schema based on the value associated with it.
+When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object can be used to aid in serialization, deserialization, and validation. The discriminator is a specific object in a schema which is used to inform the consumer of the document of an alternative schema based on the value associated with it.
 
-When using the discriminator, _inline_ schemas will not be considered.
```

Comment:
Problem 1: the verb "consider" is not conducive to precision: "to think carefully about (something), typically before making a decision." "take (something) into account when making an assessment or judgment."...

Problem 2: it’s not clear how this sentence interacts with other sentences in the spec. In particular, does the above sentence nullify the requirement for the discriminator property in the payload: 

> The expectation now is that a property with name `petType` MUST be present in the response payload.